styles/designs/webview/parts/_exercise-shapes.scss

```diff
@@ -54,6 +54,11 @@
                         $Exercise--ProblemsExercises__Solution--Before2,
                     ),
                 )),
+                map-merge($Exercise--WithProblemTitle, (
+                    _components: (
+                        $Exercise__Problem__Container--FromExerciseWithProblemTitle
+                    ),
+                )),
```

Comment:
Not the best solution, but can't be better without huge refactoring. In the future we should create separate shapes for numbered exercises, exercises with problem title and example.